docs/onboarding/azure-devops-pipelines.md

```diff
@@ -267,23 +311,25 @@ This role assignment is used to grant users access to the logging subscription b
 
 1. Edit `./config/variables/<devops-org-name>-<branch-name>.yml` in Git.  This configuration file was created in Step 3.
 
-    * Update **var-logging-managementGroupId** with the logging management group.  This is based on the prefix defined in `var-topLevelManagementGroupName`.  For example, if `var-topLevelManagementGroupName` is set to `contoso`, then `var-logging-managementGroupId` will be `contosoPlatformManagement`.
-    
+    * Update **var-logging-managementGroupId** with the logging management group:
+      * For **CanadaPubSecALZ v0.9.0 or later**, this is based on the `id` attribute of the first child element defined in your `var-managementgroup-hierarchy` configuration value. In JSON notation, this is equivalent to: `.children[0].id`.
```

Comment:
This wouldn't be the case when we have the hierarchy.  Let's rephrase it to say that with v0.9.0 or later, this will be the management id that represents the Logging Management Group in the defined hierarchy.

Reply:
Good catch; I was a little too quick with copy/paste from higher up in the document. Fixed.